src/main/java/com/turkcell/staj/dtos/packages/requests/RequestAddPackageDTO.java

```diff
@@ -1,4 +1,24 @@
 package com.turkcell.staj.dtos.packages.requests;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+
 public class RequestAddPackageDTO {
+
+    @NotBlank
+    private String packageName;
+    @NotBlank
+    private String description;
+    @Positive
```

Comment:
Burada paket fiyatının sıfır yani ücretsiz olma durumunuda ele almak için positive anno şunla değiştirilebilir.
`@PositiveOrZero(message = "Price can't be negative")`